CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 5.0.0
+### Changed
+- store configurations in a separate database table, not appconfig
+
+### Added
+- occ commands for modifying SAML configurations
```

Comment:
add section "Removed" with "ability to change SAML configuration with occ app-config, use the new occ commands instead"

this would highlight the breaking change for those using the old commands in scripting